FirebaseAdmin/FirebaseAdmin/Auth/Providers/ProviderConfigManager.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FirebaseAdmin.Util;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Http;
+
+namespace FirebaseAdmin.Auth.Providers
+{
+    /// <summary>
+    /// ProviderConfigManager provides methods for interacting with the
+    /// <a href="https://developers.google.com/identity/toolkit/web/reference/relyingparty">
+    /// Google Identity Toolkit v2</a> via its REST API. This class does not hold any mutable
+    /// state, and is thread safe.
+    /// </summary>
+    internal sealed class ProviderConfigManager : IDisposable
+    {
+        private const string ClientVersionHeader = "X-Client-Version";
+
+        private const string IdTooklitUrl = "https://identitytoolkit.googleapis.com/v2/projects/{0}";
```

Comment:
I think there's a typo and you meant `IdToolkitUrl`

Reply:
Fixed. This was copied over from `FirebaseUserManager`. I've fixed that instance too. Good catch!